app/views/admin/layouts/admin.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Groupon - Admin</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+    <%= javascript_include_tag 'application' %>
+  </head>
+
+  <header>
+    <%= render 'shared/navbar' %> 
```

Comment:
can't be same as the FE, use different navbar for admin end. admin/shared/navbar